patch/llpcPatchEntryPointMutate.cpp

```diff
@@ -781,23 +791,37 @@ FunctionType* PatchEntryPointMutate::GenerateEntryPointType(
     switch (m_shaderStage)
     {
     case ShaderStageVertex:
+    case ShaderStageTessControl:
         {
+	    // on GFX9+, LS and HS are merged into a single shader.
```

Comment:
// on GFX9+, LS and HS are merged into a single shader.
->
// NOTE: On GFX9+, Vertex shader (LS) and tessellation control shader (HS) are merged into a single shader.